### PR TITLE
Add tests for GETing a file from file-vault

### PR DESCRIPTION
**Pull request by easternbloc**
Ensures we build the correct URL for the signedUrl

### CODE REVIEW

**Comment by lennym** [comment]
How come these tests don't throw errors because of an undefined encryption password? I can't see where it's set, so I'd expect it to throw?

**Comment by lennym** [comment]
`s/undefined/changeme`

**Comment by lennym** [comment]
Cancel that, I found it in an earlier test.

https://github.com/UKHomeOffice/file-vault/blob/master/test/file.js#L19

**Comment by lennym** [comment]
You should define that variable in all of your before steps though so you don't have accidental sensitivity to execution order.

File: test/file.js [hunk 1]
```diff
@@ -8,21 +8,24 @@ const assert = require('assert');
...
+  });
+
   describe('config', () => {
-    it('returns an error if the default password isnt changed', () => {
+    it('returns an error if the default password isnt set', () => {
+      process.env.NODE_CONFIG = '{"aws": {"password":""}}';
+
       assert.throws(() => require('../controllers/file'), Error, 'please set the AWS_PASSWORD');
+
+      process.env.NODE_CONFIG = '{"aws": {"password":"atest"}}';
```
Comment (lennym) [inline test/file.js]: This would be better done in a `beforeEach`/`afterEach`. As it is `process.env` won't be reset if the test fails.

Comment (easternbloc) [reply test/file.js]: nice

Comment (easternbloc) [reply test/file.js]: will change that

Comment (easternbloc) [reply test/file.js]: updated

**Pull request merged**